scripts/create_type_file.py

```diff
@@ -161,6 +161,27 @@ def get_path_from_metadata(name, metadata_types):
     returned_path = ".".join(str(x) for x in type_with_primitive["type"]["path"])
     return returned_path
 
+def get_address_path_from_metadata(name, metadata_types):
+    data_lake = find_type_in_metadata(name, metadata_types)
+    value = metadata_types
+    value_variants = []
+    for data in data_lake:
+        value = metadata_types
+        for path in data:
+            if path in ("typeName", "name"):
+                continue
```

Comment:
Does it refer to the fields of the `CompositeField`? Which are `name`, `typeName`, `docs` and `type`?